sci-geosciences/osgearth/Manifest

```diff
@@ -1,2 +1,12 @@
+AUX osgearth-2.7-linker.patch 1322 SHA256 1d40705d04ef2e2914479aaf6a4dfae3b81c23b5cd1151082ac25fc7cf84f515 SHA512 6e42c0eca0c1b54b73a4eca1b077ac7f9be23258343504d7b81d94f95e51be11b6b91668c0b7c94e57b07bcc2132b282862b8cc21797647832b8ba07bdeaca14 WHIRLPOOL 89112c1b3c48429f8b57cf08587c2ccc0be825bcd23a3a51f201990522083d2c9d15d647e2ea10f8a6e33e056af9506c8fcd940ec6a55aeba73083ecb3fce092
 DIST osgearth-2.6.tar.gz 58605003 SHA256 965c93837520ab9538038843ff83ee1903548f8be191ee211e40abb0e6c1bb4a SHA512 3120547ef639d4c1ff0404358777317c8794a493a772dc7af11b8056fd3f799a5e16358c19970862d16f6ab98f78abb3ea25c3aa113df8a3df68d54818fb03bc WHIRLPOOL 6533e5a92ee973ff1d5bfb80ab3b051279bc65e85d00f0fb9027b6fe2be06efc5bd3907516c27e7e06b4c1cb03569650bf368b5d7728b848bc86ec9666607939
 DIST osgearth-2.7.tar.gz 61837911 SHA256 cf973b664aeb79f70e48f5cd02ba670069ec273d71fe541604ed5b328d956d83 SHA512 acc7235b2df4a226e6834f961006c35310183695925091dcdfb1339e89be8ef08773c8c62ca98eb708485e03a870dace65a27d0beb2aefc807f6e558243e4693 WHIRLPOOL 8d3afe46fbef7064235a85c78a7da160f982c7ce19d8310484feb31f0a2317de631796f471c1b547bb9a01de0958a8b28ba1d919a20be230ea4651c5fbd785a8
+DIST osgearth-2.8.tar.gz 77472117 SHA256 78120338ead2976018cc877e93342403bafe1b032f2f8b7de3915ba6c5c2486a SHA512 d7370091de178b8d8bc29f4b9c12dc8afcc18c522dc06c6b9256bda2da1d4ee00a7b2afd5161e684a5b937bb869666697ce83dbd451c02c75473db680c5ea78c WHIRLPOOL b2a638a3430258c2c03b44becd05e39cd6ffb09c0b99ec9bd08997ffd902d2e34b1deba4b7e806529bee3b06886c7a7ef63147eb38ed5acd7282bffe0f574093
+EBUILD osgearth-2.6.ebuild 1382 SHA256 448a2de5746a021f96a016d2cd8048906df31fbb9b910ca955e55b0827028b49 SHA512 80ad549f48990c99e7360f4ebfa6d2111002758d0c75f56992cf6485042633c500ce7b4a4a47609fbbf596ad72b6774cbf887868c0c80997b0c90fe452b2101f WHIRLPOOL 204e727cd9446cd4a4eb31a40f2f5c66283e8388645ad4dc01b40490b3928944fee761b7b3930042b2956bf54b8d9aaeb0ad98acc7941dba6447466392020403
+EBUILD osgearth-2.7.ebuild 1395 SHA256 b7d9f0157c90c06e4d3aa660956f5e2466e23adfd132666ea7b76447ce8e97c6 SHA512 9c85ba431ba51512adb493b9efb5f5ee62584fd66553c827623c1ff00951d5ca1f02aeee087f3d727337c6415c33227d5ad6962b3b3bb96a007c5dc057ef748d WHIRLPOOL 2469acdcd0ec34d15301708efe4a3364ef1da6e77c18f38cf44797d19f8e6ffdfd0aa1658dfd31bb517a51aac76c3983fd6c01fb404120f52f4b9ab923576b12
+EBUILD osgearth-2.8.ebuild 1391 SHA256 27e10446c4fb22d001ffdfd456a4a448d4bd7ede1c743c8895cc1c8d3c05a357 SHA512 e7f4b96ff50a885517e22de882cf9d92bb6c5a2e69a2872c4dd54e576d8c7d01a7cd52a8fbeb07b9e05c823c047caf588cd4788f023af5d6d852c4c30166d832 WHIRLPOOL 30adb567ae1c7538eaf5a4f7f01ec5c36e79b110fc6a80626823522edb0bf9858d55ba496a96e412bc013279129fbdca01e79249d4ae4efbc29b2accc0bfefb8
```

Comment:
This is a fat manifest. Please clone the repo properly and use `repoman manifest` to generate only hashes only for the tarballs.